Specification/3._Data_structure_description_language.md

```diff
@@ -197,7 +180,7 @@ Note that the use of infinity and NAN (not-a-number) is discouraged as it may no
 * Boolean `true` or `false`.
 * Single ASCII character, ASCII escape sequence, or ASCII hex literal in single quotes. E.g., `'a'`, `'\x61'`, `'\n'`.
 
-The DSDL compiler shall implicitly convert the type of an initializer expression to the constant type if
+The DSDL compiler will implicitly convert the type of an initializer expression to the constant type if
```

Comment:
I think it's a common practice to define a language by describing the expected behavior of its compiler. Look at the C++ standard, for example.

I think we should use "must" here. I thought that "shall" and "must" can be used interchangeably in this context (per RFC2119), can they not?